### PR TITLE
Calendar and Registration

### PR DESCRIPTION
**Pull request by texas-kristen**
- builds on #1631 

- creates webinar calendar page
- creates a webinar registration page
- incorporates two new pages into the tags on video items

### CODE REVIEW

**Comment by lottamus** [comment]
Deploy preview for *stoplightio* ready!

Built with commit 04300476110451dec60d26fccf9782bd9c054f6e

https://deploy-preview-1630--stoplightio.netlify.app

**Comment by lottamus** [comment]
Deploy preview for *stoplightio* ready!

Built with commit a0058c50f5c8490505e90e2db75d1bfe7c3b1f7f

https://deploy-preview-1630--stoplightio.netlify.app

**Comment by texas-kristen** [comment]
@paulatulis disregard that request for review, for the moment, I need to take the tabs off the individual videos

**Comment by paulatulis** [comment]
@texas-kristen for whatever reason, the Netlify preview is not reflecting my changes (the two boxes should be in line with each other). What does it look like on your end?
Screen Shot 2020-10-15 at 9 43 02 AM

File: src/components/Content/index.tsx [hunk 1]
```diff
@@ -4,6 +4,7 @@ import * as React from 'react';
 import { useBanner } from '../../hooks/useBanner';
 import { convertMarkdownToHTML } from '../../utils/markdown/index.js';
 import { CallToAction } from '../CallToAction';
+import { HubSpotForm, IHubSpotForm } from '../../components/HubSpotForm';
```
Comment (paulatulis) [inline src/components/Content/index.tsx:7]: Let's remove hubspot mentions from this file if we don't intend to use it right now.

File: src/templates/Subpage/index.tsx [hunk 2]
```diff
@@ -11,6 +11,7 @@ import { IQuote, Quote } from 'src/components/Quote';
 import { IRelatedPage, RelatedPages } from 'src/components/RelatedPages';
 import { Section } from 'src/components/Section';
 import { ITab } from 'src/components/Tabs';
+import { HubSpotForm, IHubSpotForm } from '../../components/HubSpotForm';
```
Comment (paulatulis) [inline src/templates/Subpage/index.tsx:14]: Same here

**Review by paulatulis — APPROVED** [review]

**Comment by texas-kristen** [comment]
@paulatulis somewhere along the way I have broken [https://deploy-preview-1630--stoplightio.netlify.app/api-design-guide-download/](https://deploy-preview-1630--stoplightio.netlify.app/api-design-guide-download/)

**Comment by paulatulis** [comment]
> @paulatulis somewhere along the way I have broken https://deploy-preview-1630--stoplightio.netlify.app/api-design-guide-download/

Good spot! I just undid the changes to the form. The Hero already accounts for tabs and all of the other props, so I think it was getting confused. 

It looks good on my local, but let's see what preview looks like on your end. Also, the video on `/ama-spectral-and-style-guides` does not exist - that should be removed from the latest list until there's a video to post, right?

**Comment by texas-kristen** [comment]
> It looks good on my local, but let's see what preview looks like on your end. Also, the video on `/ama-spectral-and-style-guides` does not exist - that should be removed from the latest list until there's a video to post, right?

@paulatulis it starts at 11, so I figure we can just hold off on merging until later today and plug it in

**Review by paulatulis — APPROVED** [review]

**Pull request merged**